pr_review_queue.py

```diff
@@ -281,13 +323,25 @@ def main():
     pull_request_list = get_pull_request_list(github_api, args.org, args.repo)
 
     if args.queue:
-        pr_review_queue = create_pr_review_queue(pull_request_list)
-        if pr_review_queue == []:
+        needs_reviewer, needs_changes, needs_review, needs_conflict_resolution = create_pr_review_queue(pull_request_list)
+        if (needs_reviewer == [] and
+            needs_changes == [] and
+            needs_review == [] and
+            needs_conflict_resolution == []):
             print("No pull requests found that match our criteria. Exiting.")
             sys.exit(0)
 
         message = ("Good morning, image builders! :meow_wave: Here are a couple of PRs :pull-request: "
-                   "that could use your :eyes:\n" + "\n".join(pr_review_queue))
+                   "that could use your :eyes:\n")
+        if needs_reviewer != []:
+            message += "---\nThese need a reviewer:\n • " + "\n • ".join(needs_reviewer)
```

Comment:
Maybe just
```suggestion
            message += "--- These need a reviewer:\n • " + "\n • ".join(needs_reviewer)
```
to be shorter, or even
```suggestion
            message += "# These need a reviewer:\n • " + "\n • ".join(needs_reviewer)
```
to look more markdown-y :-D

Reply:
While that is truly markdowny, Slack doesn't do very markdowny. Only a little.
https://api.slack.com/reference/surfaces/formatting

Also, those `---` separators were only added so the CLI output would be more structured for your review ;)